portfolio/src/main/webapp/blog_script.js

```diff
@@ -3,6 +3,22 @@
  * the blog portion of the website
  */
 
+/*
+ * Gets comments to display
+ */
+function getComments() {
+	fetch('/data?max-num=' + (document.getElementById("numComShow").value))
+        .then(response => response.json())
+        .then((comments) => {
+            const commentsElement = document.getElementById("displayComments");
+            console.log('Comments: ', comments);
+            commentsElement.innerHTML = 'Here are all the comments: <br>';
+            for (const comment of comments) {
+                commentsElement.innerHTML += comment + '<br>';
+            }
```

Comment:
It would potentially be cleaner if you can use the DOM to add your comments to the blog, but not essential.

Reply:
yeah I'll look into that! Some similar problems with the info-window